compiler/rustc_codegen_ssa/src/mir/mod.rs

```diff
@@ -375,10 +382,17 @@ fn arg_local_refs<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>>(
         .collect::<Vec<_>>();
 
     if fx.instance.def.requires_caller_location(bx.tcx()) {
+        let mir_args = if let Some(num_untupled) = num_untupled {
+            // Subtract off the tupled argument that gets 'expanded'
+            args.len() - 1 + num_untupled
+        } else {
+            args.len()
+        };
```

Comment:
```suggestion
        // Subtract off the tupled argument that gets 'expanded'
        let mir_args = num_untupled.map(|n| n -1) + args.len();
```

Reply:
This might not be a perfect analogue because these are `usize`...

Reply:
I'm not sure what you mean - 'n' will always be at least 1 (since we had an untupled argument), so there will be no underflow.